billing_5c06ad9604286304a71cfa5b.md

```diff
@@ -14,41 +13,40 @@ out to
 [customersuccess@semaphoreci.com](mailto:customersuccess@semaphoreci.com)
 (please include which type of machine you prefer).
 
-## Plans
+**Note:** If you currently have an annual subscription on Semaphore Classic and
+would like to migrate to 2.0, please reach out to
+[customersuccess@semaphoreci.com](mailto:customersuccess@semaphoreci.com).
 
-After your trial period expires, your organization will be blocked and you will
-be prompted to choose a **Free plan** or a **Paid plan**.
+## Paid plan
 
-The **Free plan** includes $10 of free credit per calendar month. At the
-beginning of each month, you will receive a new $10 credit for your
-organization. On this plan, you will be able to run 1 job and you can use the
-e1-standard-2
-[machine](https://docs.semaphoreci.com/article/20-machine-types#linux-machine-types).
-You can have an unlimited number of organizations, but only one of the
-organizations you own can be on a free plan.
-
-The **Paid plan** is activated once you add your credit card. You will be
+The Paid plan is activated once you add your credit card. You will be
 charged based on your usage at the beginning of each month for the previous
 month. Billing cycles are based on calendar months.
 
-Every organization has a set of quotas that define the maximum number of
-parallel running jobs. You can see those numbers on the
-[Quotas and limits](https://docs.semaphoreci.com/article/133-quotas-and-limits)
-page. Refer to
-[Machine types](https://docs.semaphoreci.com/article/20-machine-types)
-page for more information about available hardware you can use.
-
-If you are on a **paid plan** or in **trial** and you’d like to run more jobs in
-parallel than given by default, you can easily scale up by reaching out to
+You can find the default number of machines for Paid plans on the [Quotas and
+Limits page](https://docs.semaphoreci.com/article/133-quotas-and-limits).
+If you need to scale up, you can easily do that by going to
+**Billing > See detailed insights… > Quota > Request upgrade…** or by reaching
+out to
 [customersuccess@semaphoreci.com](mailto:customersuccess@semaphoreci.com)
-(please include which type of machine you prefer) or through the Semaphore 2.0
-UI (Billing > See detailed insights… > Quota > Request upgrade…).
+(please include which type of machine you prefer).
 
-In Semaphore 2.0 all organizations are on monthly plans.
+**Note:** In Semaphore 2.0, all organizations are on monthly plans.
 
-**Note:** If you currently have an annual subscription on Semaphore Classic and
-would like to migrate to 2.0, please reach out to
-[customersuccess@semaphoreci.com](mailto:customersuccess@semaphoreci.com).
+## Open Source plan
+
+On this plan, you have unlimited build minutes for building your public
+repositories. You also have four e1-standard-2 machines and one a1-standard-4 
+machine at your disposal.
+
+## Free plan
+
+The Free plan includes $10 of free credit per calendar month. At the
```

Comment:
Can you add: `$10 (up to 1300 build minutes)`? This is how we communicate on the pricing page. The choose plan screen also describes the free plan in terms of minutes.